src/aead/aes.rs

```diff
@@ -136,10 +136,16 @@ unsafe fn ctr32_encrypt_blocks(
     let (input, leftover) = slice::as_chunks(&in_out[src]);
     debug_assert_eq!(leftover.len(), 0);
 
-    let blocks = input.len();
+    let blocks = match NonZeroUsize::new(input.len()) {
+        Some(blocks) => blocks,
+        None => {
+            return;
```

Comment:
This is covered by tests, but only when NEON is enabled on 32-bit ARM, when BSAES falls back to VPAES. Unfortunately, we don't have code coverage measurement for 32-bit ARM since the Rust toolchain doesn't enable the necessary infrastructure for it for these targets.